packages/client/src/pages/PortalSettings/categories/integration/ThirdPartyServicesSettings/sub-components/consumerModalDialog.js

```diff
@@ -246,6 +250,7 @@ class ConsumerModalDialog extends React.Component {
           <Button
             size="normal"
             scale
+            id="cancel"
```

Comment:
Too simple id value (potential duplicates)

Reply:
Fix to "cancel-button"